docs/release-procedure.md

```diff
@@ -121,3 +121,13 @@ Fore more information please check [Helm documentation](https://helm.sh/docs/top
 - upload the packaged chart to the release in this repository
 - update the [index.yaml](https://github.com/apache/incubator-yunikorn-release/blob/gh-pages/index.yaml) file in the gh-pages branch with the new release
 
+## Update the website
+
+- Create a new version of the [YuniKorn website](https://github.com/apache/incubator-yunikorn-site/tree/master).
```

Comment:
I think this is better to say "Create a new documentation version on YuniKorn website based on the latest content in [docs](https://github.com/apache/incubator-yunikorn-site/tree/master/docs) directory."
Pls link to https://github.com/apache/incubator-yunikorn-site/tree/master#release-a-new-version

Reply:
Added the sentence and the link